functions.php

```diff
@@ -30,7 +30,7 @@ function add_section_markup_before( $content, $section, $class, $title, $section
 	// Retrieve background image sizes
 	$bg_image_sm_id = get_field( 'section_background_image', $section->ID );    // -sm+
```

Comment:
If this is now the `lg` image, let's just go ahead and update the variable name so it's obvious what's happening.

Reply:
Variable name was changed to $bg_image_lg_id